### ISSUE TITLE
Wrap the Hyrax header in a <header> tag

### ISSUE THREAD

**Issue by adamjarling** [samvera/hyrax#3082]
### Descriptive summary

From the Accessibility360 Audit Report:

> Wrap the header in a <header> tag.

> These pages require ARIA landmarks to assist screen reader users in understanding and navigating the structure of the pages.

> Affects all pages on the site.

#### Severity Level
Serious

![header-tag](https://user-images.githubusercontent.com/3020266/40318572-a0a048a4-5cea-11e8-8b18-0a7a49e1371a.png)

### Rationale

These pages require ARIA landmarks to assist screen reader users in understanding and navigating the structure of the pages.

### Expected behavior
Home top stripe (see attached image) containing navigation links and logo should be wrapped in a `<header>` tag.

### Actual behavior
No `<header>` tag exists (see attached image Elements tab).
